docs/naming_conventions.md

```diff
@@ -0,0 +1,52 @@
+The platform impose several constraints with regards to the naming of the persistence elements (files, databases, tables and collections). Such constraints, and the way the different storages solve them, must be had into account when designing the data model of the solution running on top of the platform.
+
+#Global character set and size limitations
+Globally, the platform impose certain rules when defining the data model. These rules directly come from Orion Context Broker [constraints]((http://fiware-orion.readthedocs.io/en/develop/user/forbidden_characters/index.html)) regarding the character set used for entity and attribute names and their types: 
+
+* The following charaters are forbidden: `<`, `>`, `"`, `'`, `=`, `;`, `(` and `)`.
+* The following ones should also be avoided: white space, `?`, `/`, `%` and `&`.
+
+Additionally, Orion Context Broker imposes the following restrictions:
```

Comment:
A reference to the document where the multitenancy model is described, along with service and service path concept, should be included: multitenancy.md. In fact, the restrictions described in naming_conventions.md for service and service path are reducnant with the ones in multitenancy.md. To avoid future inconsistences due to manually duplicating the same description in two places, probably they should be unified.